### ISSUE TITLE
Welcome Tour: Improve tour close out

### ISSUE THREAD

**Issue by chriskmnds** [Automattic/wp-calypso#56168]
### What

Better shortcuts for closing out the tour.

### Why

Currently for a user to close out the tour they'd need to first open it into the expanded view, then find the respective X control. This would mean a couple of clicks, while expanding something to then dismiss it may not be the friendliest approach, especially in a keyboard environment. 

### How

Ideas to explore:

1. Esc could close/dismiss the tour entirely when minimized and focused.
2. Introduce an X button next to the expand button in the minimized view (similar to notices). 
3. Signifying the Esc functionality on the open tour (when using keyboard to focus/navigate) as discussed in p1631278866058600/1631274609.056600-slack-C029WSAQMT9

**Comment by chriskmnds** [comment]
@poligilad-auto - I just put together in the description some of the ideas by @razvanpapadopol mainly in Slack: p1631275988057400/1631274609.056600-slack-C029WSAQMT9 We can add more if anything else comes up in the discussion.

I believe the design input/check right now would be around point 2 above (adding an X button to the minimized view)? If we agree on the direction. The whole block is clickable currently and opens the portal.

<img width="318" alt="Screenshot 2021-09-10 at 4 54 34 PM" src="https://user-images.githubusercontent.com/1705499/132864558-89874adb-2cd7-42ae-ac92-68d3728fcfa9.png">

**Comment by poligilad-auto** [comment]
Thanks @chriskmnds. We could do something like this, like @razvanpapadopol suggested.

![image](https://user-images.githubusercontent.com/83961704/133071586-3235f4d9-e95b-4d7e-861e-503d1b4b93e7.png)

**Comment by chriskmnds** [comment]
Thanks @poligilad-auto . Looks good! The only other bit to consider is the hoverable behavior in the block I think. Right now it's a single button that gets highlighted when hovered. I guess it's a tiny detail. Do we leave an empty/non-reactive space between the expand and close (X) areas, or one continuous flow?

![Kapture 2021-09-13 at 15 58 11](https://user-images.githubusercontent.com/1705499/133088109-8f087e42-9e51-4ced-a17c-1a5668a07f07.gif)

**Comment by razvanpapadopol** [comment]
By checking an actionable notice in Calypso, I see there's no space between the two buttons (areas). Where "View your cart" link area ends, the (X) dismiss button area starts. They are also displayed without any visual separator in between.

<img width="967" alt="Screenshot 2021-09-13 at 16 17 19" src="https://user-images.githubusercontent.com/14192054/133090511-f11b3af7-a41c-48fb-a219-5f31a35d5f77.png">

So I'd go with the version shared by @poligilad-auto [above](https://github.com/Automattic/wp-calypso/issues/56168#issuecomment-918112912) with just a bit more spacing (20-24px) between the expand icon and the close icon. Right now there is a 12px horizontal padding on the CTA button so we probably want the same or slightly less for the close button.

<img width="571" alt="Screenshot 2021-09-13 at 16 37 01" src="https://user-images.githubusercontent.com/14192054/133093962-0b6568ca-08ad-4bf9-a188-0583432c869c.png">

**Comment by chriskmnds** [comment]
Makes sense Razvan. Leaving a bit of dead space in between would actually look a little weird/awkward. I think my point in general is whether we'd want any better indication/effect as we hover over one icon and onto the next. In Calypso they play with opacity/color a little and it's pretty clear. In our case we change the color of the text to blue, which is further to the left, while the icon remains static. but maybe just the color of the text changing is enough.

![foo](https://user-images.githubusercontent.com/1705499/133269443-bc0404d6-41c4-4fda-9b67-9d0bf9f6741e.gif)

**Comment by chriskmnds** [comment]
Only last point left here to address:

> Signifying the Esc functionality on the open tour (when using keyboard to focus/navigate)
